supabase/lib/client_options.py

```diff
@@ -51,6 +51,8 @@ class ClientOptions:
     flow_type: AuthFlowType = "implicit"
     """flow type to use for authentication"""
 
+    access_token: Union[str, None] = None
```

Comment:
@silentworks this needs to be a function that asynchronously return the access token.

https://github.com/supabase/supabase-js/pull/1004/files#diff-b8b77f5be18cf56dca425b3a5b8e9d2e754dd37fe0e3612b95cd4e9bccda08a5R80